server/api/models/story.js

```diff
@@ -0,0 +1,38 @@
+const pool = require('../dbConfig/init');
+const db = require('../dbConfig/init');
+
+class Story {
+    constructor(data) {
+        this.title = data.title;
+        this.authorName = data.authorName;
+        this.story = data.story;
+    }
+
+    static async create(storyData) {
+        return new Promise(async(resolve, reject) => {
+            try {
+                const { title, authorName, story } = storyData;
+                let result = await db.query('INSERT INTO storyDatabase (title, authorName, story) VALUES ($1, $2, $3) RETURNING *;', [title, authorName, story]);
+                resolve(pool.query(result));
```

Comment:
I thought the result here is already making a query to the SQL database on line 15, what does the pool.query(result) do?